pyproject.toml

```diff
@@ -28,6 +28,7 @@ dependencies = [
     "pycognito==2024.5.1",
     "PyJWT>=2.8.0",
     "snitun==0.39.1",
+    "webrtc-models==0.1.0",
```

Comment:
Can we make this more loose? like <1?
How strict is mashumaro? If the lib is updated to add more attributes, will the implementation here break?

Reply:
Yes, I think we can. Considering we create the `RTCIceServer` manually and pass parameters to it, as long as the new attributes are optional, the implementation will not break.